src/assets/css/librarybrowser.css

```diff
@@ -134,19 +153,26 @@
 
 .headerLeft {
     display: flex;
+    -webkit-align-items: center;
     align-items: center;
+    -webkit-box-flex: 1;
+    -webkit-flex-grow: 1;
     flex-grow: 1;
     overflow: hidden;
-    justify-content: left;
```

Comment:
why remove?

Reply:
These are all bringing back these to how they looked pre-removal.

What I did is basically revert the merge commit that pulled the PR in the history. So everything gets reset to how it was pre-merge.

I'll go over the changes to fix these once I'm home.